mmv1/products/gkeonprem/VmwareNodePool.yaml

```diff
@@ -0,0 +1,286 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+name: "VmwareNodePool"
+min_version: beta
+base_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools"
+create_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools?vmware_node_pool_id={{name}}"
+update_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools/{{name}}"
+self_link: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools/{{name}}"
+update_verb: :PATCH
+update_mask: true
+description: "A Google Vmware Node Pool."
+autogen_async: false
+id_format: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools/{{name}}"
+import_format: ["projects/{{project}}/locations/{{location}}/vmwareClusters/{{vmware_cluster}}/vmwareNodePools/{{name}}"]
+taint_resource_on_failed_create: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 60
+  delete_minutes: 60
+  update_minutes: 60
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "gkeonprem_vmware_node_pool_basic"
+    min_version: beta
+    primary_resource_id: "nodepool-basic"
+    vars:
+      name: "nodepool"
+    test_env_vars:
+      project: "fake-backend-360322"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "gkeonprem_vmware_node_pool_full"
+    min_version: beta
+    primary_resource_id: "nodepool-full"
+    vars:
+      name: "nodepool"
+    test_env_vars:
+      project: "fake-backend-360322"
+skip_sweeper: true
```

Comment:
Why are we skipping sweeper generation?

Reply:
My understanding is sweeper generation can be skipped for child resources. Node pool is a child resource of cluster. Please help correct if I am wrong.

Reply:
My understanding is sweeper generation can be skipped for child resources. Node pool is a child resource of cluster. Please help correct if I am wrong.

Reply:
It _can_ be skipped, but we should generate them if we're able to.

Reply:
Thanks for the guidance. Removed it to generate sweeper